src/ui/public/notify/notifier.js

```diff
@@ -97,10 +97,14 @@ function add(notif, cb) {
     });
   } else if (notif.customActions) {
     // wrap all of the custom functions in a close
-    notif.customActions = notif.customActions.map(action => {
+    notif.customActions = notif.customActions.map((action, idx) => {
       return {
         key: action.text,
-        callback: closeNotif(notif, action.callback, action.text)
+        callback: closeNotif(notif, action.callback, action.text),
+        getButtonClass() {
+          if (idx === 0) return `btn-primary btn-${notif.type}`;
+          return `btn-default btn-${notif.type}`;
+        }
```

Comment:
I like the suggestion, but I will restructure this so that we can use all `const` and there aren't many repeated assignments in the `.map`. I'll create a function outside of `function add() {}` that returns the className based on it's input param.